### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry): closed immersions are local at the target

### PR DESCRIPTION
**Pull request by chrisflav**
Shows that closed immersions are local at the target.

---

I hope this does not cause too much friction with #14430.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/AffineScheme.lean [hunk 1]
```diff
@@ -121,6 +121,13 @@ theorem isAffine_of_isIso {X Y : Scheme} (f : X ⟶ Y) [IsIso f] [h : IsAffine Y
   rw [← mem_Spec_essImage] at h ⊢; exact Functor.essImage.ofIso (asIso f).symm h
 #align algebraic_geometry.is_affine_of_iso AlgebraicGeometry.isAffine_of_isIso
 
+/-- If `f : X ⟶ Y` is a morphism between affine schemes, the corresponding arrow is isomorphic
+to the by the map on global sections induced morphism on prime spectra. -/
```
Comment (semorrison) [inline Mathlib/AlgebraicGeometry/AffineScheme.lean:125]: Typo?

**Comment by semorrison** [comment]
bors d+ erdOne

**Comment by semorrison** [comment]
Oops, wrong bors syntax. My intent was to let @erdOne sign off on this on case of conflicts with their refactors.

**Comment by erdOne** [comment]
I hope we can get #14430 in first unless you think it needs more work.

**Comment by chrisflav** [comment]
> I hope we can get #14430 in first unless you think it needs more work.

I agree, will have a final look at #14430.

File: Mathlib/AlgebraicGeometry/AffineScheme.lean (hunk 1 above)
Comment (chrisflav) [reply Mathlib/AlgebraicGeometry/AffineScheme.lean]: I changed the wording.

**Comment by chrisflav** [comment]
I merged your refactor @erdOne. Are you happy with this PR?

**Comment by erdOne** [comment]
Yes. Thanks! Please merge it once it passes CI.

**Comment by chrisflav** [comment]
Thanks for the reviews!

bors r+

**Pull request closed without merging**